domains/varun.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Devloper Portfolio",
+  "repo": "https://github.com/varun-chaudhary/portfolio.github.io",
+  "owner": {
+    "username": "varun-chaudhary",
+    "email": "varunc105@gmail.com"
+  },
+  "record": {
+    "CNAME": "portfolio.github.io"
```

invalid CNAME record
```suggestion
    "CNAME": "varun-chaudhary.github.io"
```